src/index.html

```diff
@@ -5,10 +5,35 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <a class="header__logo" href="#"> <img src="/src/images/logo.png" alt="logo"></a>
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item"> <a class="nav__link is-active" href="#apple">Apple</a> </li>
+            <li class="nav__item"> <a class="nav__link"  href="#samsung">Samsung</a> </li>
+            <li class="nav__item"> <a class="nav__link" href="#smartphones">Smartphones</a> </li>
```

Comment:
- Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/211fb52c-7b85-4ab0-9638-cba3fbbe2b80)